<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="Label_avatar" name="lab_avatar">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:transparent;
border: 2px solid  white;
border-radius:25px;
}
QLabel:hover{
background:transparent;
border: 2px solid  black;
border-radius:25px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
      <widget class="QToolButton" name="toolButton_5">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
background:transparent;
border-radius :15px;
qproperty-icon:url(:/images/help.png);
qproperty-iconAlignment: AlignCenter;
qproperty-iconSize:30px 30px;
}



QToolButton:hover{
background:rgb(230, 230, 230);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lab_logo">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_logo_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>赛搏</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="LineEdit" name="linee_search">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;
border-radius: 3px;
background:rgb(250, 250, 250);
color:grey;
padding: 5px;
</string>
     </property>
     <property name="text">
      <string>请输入比赛名称/比赛ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolbu_search">
     <property name="styleSheet">
      <string notr="true">QToolButton {
    border: none;
    background: transparent;
	image :url(:/images/search_blue.svg);
}

QToolButton:hover{
	image :url(:/images/search_black.svg);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>452</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolbu_min">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
    border: none;
    background: transparent;
	qproperty-icon: url(:/images/minmize.svg);  
    qproperty-iconSize: 19px 20px;
    color: rgb(78, 78, 78);
    font:  16pt 'Microsoft New Tai Lue';
}

QToolButton:hover {
    border: none;
    background: transparent;
    font:  16pt 'Microsoft New Tai Lue';
background: rgb(234,235,237);
    border-radius: 3px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolbu_max">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border:none;
	qproperty-icon:url(:/images/maxmize.svg);
	qproperty-iconSize:14px 17px;
}
QToolButton:hover{
	border:none;
background: rgb(234,235,237);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolbu_close">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
qproperty-icon:url(:/images/close_b.png);
qproperty-iconSize:14px  20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Label_time" name="lab_time">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time______</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="Label_time" name="lab_date">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pu_facebind">
     <property name="text">
      <string>人脸未绑定（点击进行绑定）</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="ListView" name="listView_game_detail"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>rewritten_widgets/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Label_time</class>
   <extends>QLabel</extends>
   <header location="global">rewritten_widgets/label.h</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header location="global">rewritten_widgets/listview.h</header>
  </customwidget>
  <customwidget>
   <class>Label_avatar</class>
   <extends>QLabel</extends>
   <header>rewritten_widgets/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
